docs/UserGuide.md

```diff
@@ -387,7 +374,7 @@ This rate will be used in the calculation of fees due after each lesson.
 
 Adds a lesson to the specified student in TAB.
 
-Format: `ladd INDEX [recurring/[END_DATE]] date/dd MMM yyyy time/HHmm-HHmm subject/SUBJECT rates/LESSON_RATES [hw/HOMEWORK]…​`
+Format: `ladd INDEX [recurring/[END_DATE]] date/dd MMM yyyy time/HHmm-HHmm subject/SUBJECT rates/LESSON_RATES [f/OUTSTANDING_FEES] [hw/HOMEWORK]…​`
```

Comment:
I think no need to wrap end date with brackets. its consistent with the behaviour of every other optional field
```suggestion
Format: `ladd INDEX [recurring/END_DATE] date/dd MMM yyyy time/HHmm-HHmm subject/SUBJECT rates/LESSON_RATES [f/OUTSTANDING_FEES] [hw/HOMEWORK]…​`
```